test/smoke/rocmaster_check_smoke.sh

```diff
@@ -5,6 +5,11 @@
 #
 #
 
+echo "This script is deprecated. Please run : cd aomp/bin; ./run_rocm_test.sh"
```

Comment:
Did you want to wrap this in a variable check so you can continue with the script? By default I think the script should exit, but for our dev purposes we can override it if need be.